docs/_sources/index.rst.txt

```diff
@@ -51,9 +51,27 @@ The source code is hosted on `GitHub`_.
 Bug Reports
 ===========
 
-You can send feedback, bug reports and feature requests to mabwiser@fmr.com.
+You can send feedback to mabwiser@fmr.com. Please use the GitHub `Issues`_ tracking for bug reports and feature requests.
+
+Citation
+========
+You can cite MABWiser as:
+
+E. Strong,  B. Kleynhans, and S. Kadioglu, "MABWiser: A Parallelizable Contextual Multi-Armed Bandit Library for Python," in 2019 IEEE 31st International Conference on Tools with Artificial Intelligence (ICTAI 2019) (pp.885-890). IEEE, 2019.
+
+.. code-block:: bibtex
```

Comment:
I like this one! This can be repeated in README I believe. The easier/obvious to cite, the better. Plus having the paper upfront/explicitly on the landing page is a good sign for the library